source/Calamari.Tests/Fixtures/PackageRetention/JsonJournalRepositoryFixture.cs

```diff
@@ -38,14 +39,14 @@ public void WhenTheJournalIsLoadedAndCommittedTo_ThenTheJournalContainsTheCorrec
         {
             var journalPath = packageRetentionJournalPath == null ? null : Path.Combine(testDir, packageRetentionJournalPath);
 
-            var variables = Substitute.For<IVariables>();
-            variables.Get(KnownVariables.Calamari.PackageRetentionJournalPath).Returns(journalPath);
-            variables.Get(TentacleVariables.Agent.TentacleHome).Returns(testDir);
+            var variables = new CalamariVariables();
+            variables.Set(KnownVariables.Calamari.PackageRetentionJournalPath, journalPath);
+            variables.Set(TentacleVariables.Agent.TentacleHome, testDir);
```

Comment:
I decided to use `CalamariVariables` instead of a mock since this is what the other tests do. It's also more realistic since it's a proper variable bag rather than a fake one.